database/factories/ModelFactory.php

```diff
@@ -54,6 +55,7 @@
         'quantity_pending' => $faker->randomNumber(4),
         'why_participated' => $faker->sentence(),
         'source' => 'phoenix-web',
+        'details' => $details_options[rand(0, 3)],
```

Comment:
I think you can use `faker->randomElement($details_options)`;

Reply:
GREAT FIND!!!!!